admin/scripts/common/models/__init__.py

```diff
@@ -0,0 +1,88 @@
+from enum import Enum
+from typing import Any, Optional
+
+from pydantic import BaseModel, Field, field_validator, model_validator
+
+
+class Choice(str, Enum):
+    yes = "Yes"
+    not_planned = "Not planned"
+    not_yet = "Not yet"
+
+
+class Choice2(str, Enum):
+    yes = "Yes"
+    not_sure_yet = "Not sure yet"
+    no = "No"
```

Comment:
Does it make sense to have these as an `Option[bool]`?

- yes -> True
- not_planned/not_sure -> None
- no/not_yet -> False